src/iframe-helper.js

```diff
@@ -416,8 +416,12 @@ export class SubscriptionApi {
    * @param {boolean} is3p set to true if the iframe is 3p.
    * @param {function(!JsonObject, !Window, string)} requestCallback Callback
    *     invoked whenever a new window subscribes.
```

Comment:
adding sandbox="allow-scripts allow-same-origin" might solve your problem. postMessage to `*` can be a security hole, in case hacker redirected the iframe location to his own host after the handshake, then they start to receive data.